### ISSUE TITLE
Couldn't resize NGUI

### ISSUE THREAD

**Issue by Butnine** [bbepis/XUnity.AutoTranslator#570]
![20240831033908_1](https://github.com/user-attachments/assets/565f5629-830e-4531-af5b-ff76a12468e7)
I'd like to increase the size of this text in the screenshot.

![BepInEx](https://github.com/user-attachments/assets/5d7aae9c-c1a6-4549-aede-95b9d0cbcead)
So, in the BepInEx console, I figured out the Text Path as shown above.

Then I created a resizer.txt file in the BepInEx\Translation\ko\Text directory and wrote the following line
`/Main/UI Root/DialogManager/Container/MaskText/Text=ChangeFontSizeByPercentage(1.9)`

And I ran the game again, but there is no change, I need help.

**Comment by ManlyMarco** [comment]
Did you place it in a file with name that ends with `resizer.txt`? You also might need to restart the game to see effects.

**Comment by Butnine** [comment]
Yes I initially created the resizer file with the name `resizer.txt`, and now I've renamed it `Dialog_resizer.txt` and tried again, but it doesn't work.

And every time I change it, I restart the game.

By any chance, is there an option in Config.ini that I need to change for resizer.txt to work?
